terraform/environments/development/animal/vars

```diff
@@ -0,0 +1,11 @@
+# ------------------------------------------------------------------------------
+# Environment
+# ------------------------------------------------------------------------------
+
+payment_reconciler_bucket = "payment_reconciler"
```

Comment:
It doesn't look like this bucket has been created? This will need to be provisioned too. In addition the naming convention is for hyphens rather than under scores i.e. `payment-reconciler`.
What is this used for? It looks to have been defined as a variable in a few places without being used.